### DISCUSSION TITLE
Convert PNG to ICO?

### DISCUSSION THREAD

**Discussion by katastrophe92** [dlemstra/Magick.NET#1372] [General]
Hello Magick community! I have just a quick question: can this software convert .png to .ico? Hope someone can help me. Thanks!

**Comment by dlemstra** [comment]
Yes that is possible.

**Comment by katastrophe92** [comment]
Hello @dlemstra !  it is possible to convert a folder content or multiple .png files at once (in batch) what do I need to type? Thanks 👍

**Reply by dlemstra** [reply]
You will need to iterate through the folder and create a `MagickImage` for each file and then save it as an ico file. I will leave the exact code as an exercise for the reader.
